src/ocm/components/featureSupportLevels/featureStateUtils.ts

```diff
@@ -92,16 +94,6 @@ const getOdfDisabledReason = (cluster: Cluster | undefined, isSupported: boolean
   return undefined;
 };
 
-const getLvmDisabledReason = (cluster: Cluster | undefined, isSupported: boolean) => {
```

Comment:
Removing this condition seems wrong.
For clusters using < 4.11, LVM should be shown, but should be disabled as it's unsupported.
We must also put back the double if in `LvmCheckbox`

Reply:
I'll keep this removed. For <4.11 the checkbox will be hidden based on `/feature-support-levels` and this function will not be used anywhere.